src/contrib/dependencyinjection/Akka.DI.Core/DIExt.cs

```diff
@@ -38,7 +38,7 @@ public void Initialize(IDependencyResolver dependencyResolver)
         /// <returns>A <see cref="Akka.Actor.Props"/> configuration object for the given actor type.</returns>
         public Props Props(Type actorType)
         {
-            return new Props(typeof(DIActorProducer), new object[] { dependencyResolver, actorType });
```

Comment:
If this was the only place where the old DIActorProducer got with `new object[] { dependencyResolver, actorType }` created, then we could remove the obsolete `Props.CreateProducer` method.

Reply:
how do we mark deletion in version?
simply put some text in the obsolete attribute message?

Reply:
Correct - just mention that in the `ObsoleteAttribute`'s text